tap_gorgias/client.py

```diff
@@ -23,10 +25,11 @@ def get(self, url):
             url = f'{self.base_url}{url}'
 
         for num_retries in range(self.MAX_RETRIES):
-            LOGGER.info(f'gorgias get request {url}')
+            LOGGER.info(f'gorgias get request {url}, timeout={DEFAULT_TIMEOUT}')
             resp = requests.get(
                 url,
-                auth=(self.username, self.password)
+                auth=(self.username, self.password),
+                timeout=DEFAULT_TIMEOUT
```

Comment:
Also not in the diff, but `post` should also have timeout?

Reply:
good catch! thanks!